region/crateria/west/Gauntlet Entrance.json

```diff
@@ -228,24 +228,63 @@
       "link": [1, 2],
       "name": "Base",
       "requires": [
-        "ScrewAttack"
+        "ScrewAttack",
+        {"or": [
+          "canCarefulJump",
+          {"acidFrames": 15}
+        ]}
       ],
       "clearsObstacles": ["A"]
     },
     {
       "link": [1, 2],
       "name": "Gauntlet Entrance with Bombs (Left to Right)",
       "requires": [
-        "h_canUseMorphBombs"
+        "h_canUseMorphBombs",
+        {"or": [
+          "canTrickyJump",
+          {"ammo": {"type": "Super", "count": 2}},
+          {"acidFrames": 100}
+        ]},
+        {"or": [
+          "canDodgeWhileShooting",
+          "Wave",
```

Comment:
Spazer feels like a fine alternative here. It doesnt (usually) go through the bomb blocks, but it is very easy to aim.